### PR TITLE
eos: Added new states to eos_l2_interfaces

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS gomathiselvi@gmail.com

SUMMARY
Related to #23

Add gathered , parsed and rendered to eos_l2_nterfaces

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
eos_l2_interfaces.py

### CODE REVIEW

File: plugins/modules/eos_l2_interfaces.py [hunk 1]
```diff
@@ -84,17 +84,30 @@
...
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the NX-OS device by executing
+        the command B(show running-config | section ^interface).
+      - The state I(parsed) reads the configuration from C(running_config) option and transforms
+        it into Ansible structured data as per the resource module's argspec and the value is then
+        returned in the I(parsed) key within the result.
+    type: str
+    version_added: "1.0.0"
```
Comment (rohitthakur2590) [inline plugins/modules/eos_l2_interfaces.py:101]: IMO this should be only on module.

**Comment by GomathiselviS** [comment]
recheck

File: plugins/modules/eos_l2_interfaces.py [hunk 2]
```diff
@@ -84,17 +85,28 @@
...
           as changed=True.
-        version_added: '2.10'
         type: str
         choices:
         - access
         - trunk
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the NX-OS device by executing
```
Comment (NilashishC) [inline plugins/modules/eos_l2_interfaces.py:95]:
```suggestion
      - The value of this option should be the output received from the EOS device by executing
```

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**